test/oqs_test_evp_pkey_params.c

```diff
@@ -400,7 +400,7 @@ int main(int argc, char **argv) {
     const OSSL_ALGORITHM *algs;
     int query_nocache;
     int errcnt;
-    int ret = EXIT_FAILURE;
+    int test = 0;
```

Comment:
What's the reason for this variable? It looks like it's never changed: Can it be removed for better code readability?